### PR TITLE
feat(pandar_description): add pandar_qt128 description

### PR DESCRIPTION
**Pull request by TomohitoAndo**
## Description
I added sensor description of Pandar QT128

Related:
[TIER IV INTERNAL
](https://tier4.atlassian.net/browse/RT0-31480)
https://www.hesaitech.com/downloads/#qt128

image
image

## Tests performed

image

image

image

### CODE REVIEW

File: pandar_description/urdf/pandar_qt128.xacro [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<robot name="pandar_qt128" xmlns:xacro="http://ros.org/wiki/xacro">
+  <!-- Define materials -->
+  <material name="gray_material">
+    <color rgba="0.9 0.9 0.9 1.0"/>
+  </material>
+
+  <xacro:macro name="PandarQT-128" params="name parent x y z roll pitch yaw">
```
Comment (TomohitoAndo) [inline pandar_description/urdf/pandar_qt128.xacro:8]: `PandarQT-128`の命名はPandarOT-128に合わせた
https://github.com/tier4/sensor_component_description/blob/cf6acf3c03ed9ae633048e5eec3c6fc1ac660946/pandar_description/urdf/pandar_ot128.xacro#L3

File: pandar_description/urdf/pandar_qt128.xacro [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      </visual>
+      <!-- LiDAR connector visualization -->
+      <visual>
+        <geometry>
+          <cylinder length="0.015" radius="0.0057"/>
+        </geometry>
+        <origin xyz="0 0.051 0.0144" rpy="1.57 0 0"/>
+        <material name="gray_material"/>
+      </visual>
+    </link>
```
Comment (TomohitoAndo) [inline pandar_description/urdf/pandar_qt128.xacro:54]: `<inertial>` has meaningless value currently, and this causes following ERROR. So I removed.
`[ERROR 1720051304.540860749] [rviz2]: The link is static or has unrealistic inertia, so the equivalent inertia box will not be shown.`

https://github.com/tier4/sensor_component_description/blob/cf6acf3c03ed9ae633048e5eec3c6fc1ac660946/pandar_description/urdf/pandar_qt.xacro#L55-L59

File: pandar_description/urdf/pandar_qt128.xacro [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        <geometry>
+          <cylinder length="0.007" radius="0.043"/>
+        </geometry>
+        <origin xyz="0 0 0.08" rpy="0 0 0"/>
+        <material name="gray_material"/>
+      </visual>
+      <!-- LiDAR connector visualization -->
+      <visual>
+        <geometry>
+          <cylinder length="0.015" radius="0.0057"/>
```
Comment (TomohitoAndo) [inline pandar_description/urdf/pandar_qt128.xacro:49]: The connector is actually ellipse, but simply define as cylinder here.

**Review by 0x126 — APPROVED** [review]
LGTM

**Pull request merged**